<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileNameEditWithBrowse</class>
 <widget class="QWidget" name="FileNameEditWithBrowse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>28</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="fileNameEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="browseButton">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
